Services/Concurrency/RateLimiting.cs

```diff
@@ -97,6 +97,6 @@ public long GetPauseForNextMessage()
         /// <summary>
         /// Get message throughput (messages per second)
         /// </summary>
-        public double GetThroughputForMessages() => this.messaging.GetThroughputForMessages();
+        public double GetThroughputForMessages() => System.Math.Round(this.messaging.GetThroughputForMessages(), 2);
```

Comment:
could you use Math.Ceiling instead?

* Math.Round(0.005, 2) => 0
* Math.Ceiling(0.005 * 100) / 100 => 0.01